src/main/java/org/jabref/logic/formatter/casechanger/CamelNFormatter.java

```diff
@@ -0,0 +1,48 @@
+package org.jabref.logic.formatter.casechanger;
+
+import java.util.stream.Collectors;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class CamelNFormatter extends Formatter {
+    private final int length;
+
+    public CamelNFormatter(int length) {
+        this.length = length;
+    }
+
+    @Override
+    public String getName() {
+        return Localization.lang("Cameln");
```

Comment:
```suggestion
        return Localization.lang("Camel case - n letters max");
```

Reply:
sure i can work on the tests u mentioned but I would keep the ones I have implemented